test/test_utils.hpp

```diff
@@ -23,16 +23,37 @@
 #include <sempr/core/IncrementalIDGeneration.hpp>
 
 // geometries
+#include <sempr/entity/spatial/Geometry.hpp>
 #include <Geometry_odb.h>
+
+#include <sempr/entity/spatial/GeometryCollection.hpp>
 #include <GeometryCollection_odb.h>
+
+#include <sempr/entity/spatial/Point.hpp>
 #include <Point_odb.h>
+
+#include <sempr/entity/spatial/LineString.hpp>
 #include <LineString_odb.h>
+
+#include <sempr/entity/spatial/LinearRing.hpp>
+#include <LinearRing_odb.h>
+
+#include <sempr/entity/spatial/Polygon.hpp>
 #include <Polygon_odb.h>
 
+#include <sempr/entity/spatial/MultiPoint.hpp>
```

Comment:
Just a side note: `<MultiPoint_odb.h>` automatically includes `<sempr/entity/spatial/MultiPoint.hpp>`, which is why I left some of these includes out. :wink: